cuqi/sampler/_mh.py

```diff
@@ -88,7 +88,7 @@ def _sample(self, N, Nb):
 
         # initial state    
         samples[:, 0] = self.x0
-        target_eval[0] = self.target.logpdf(self.x0)
+        target_eval[0] = self.target.logd(self.x0)
```

Comment:
If the target density has a log probability density function it would still work. But some may not have that in the future. Also for e.g. `Posterior` we really did not have the logpdf, only the logd